delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/Acls.scala

```diff
@@ -301,4 +295,40 @@ object Acls {
       case c: DeleteAcl   => delete(c)
     }
   }
+
+  private[delta] def fetchWithAncestors(address: AclAddress, acls: Acls, perms: Permissions): UIO[AclCollection] = {
+    def collectionFor(addr: AclAddress): UIO[AclCollection] =
+      acls.fetch(addr).map {
+        case Some(value) => AclCollection(value)
+        case None        => AclCollection()
+      }
+
+    def rootCollection: UIO[AclCollection] =
+      acls.fetch(AclAddress.Root).flatMap {
+        case Some(value) => UIO.pure(AclCollection(value))
+        case None        =>
+          perms.fetchPermissionSet.map(ps =>
+            AclCollection(
+              Current(
+                acl = Acl(AclAddress.Root, Identity.Anonymous -> ps),
+                rev = 0L,
+                createdAt = Instant.EPOCH,
+                createdBy = Identity.Anonymous,
```

Comment:
Why anonymous? Wasn't this chosen by configuration?

Reply:
it was always anonymous; I also don't understand the value of having anything different from Anonymous driven by config, since the API can be used to grant any custom access.